src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValue.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.PowerFx.Types
     /// Represent a value in the formula expression. 
     /// </summary>
     [DebuggerDisplay("{ToObject().ToString()} ({Type})")]
+    [ThreadSafeImmutable]
```

Comment:
>   [ThreadSafeImmutable] [](http://example.com/codeflow?start=1&length=24)

I think there are cases where this (notably Table/Record) can get mutated. Is there a case forcing this?

Reply:
I don't remember which one require this one be immutable. I remove this out of PR.